vmdk_plugin/drivers/vmdk/vmdk_driver.go

```diff
@@ -177,33 +172,36 @@ func (d *VolumeDriver) MountVolume(name string, fstype string, id string, isRead
 		return mountpoint, err
 	}
 
-	watcher, skipInotify := fs.DevAttachWaitPrep(name, watchPath)
-
-	// Have ESX attach the disk
-	dev, err := d.ops.Attach(name, nil)
-	if err != nil {
-		return mountpoint, err
+	watchCtx, errWatch := fs.DevAttachWaitPrep()
```

Comment:
There are too many `err*` variables in that func, so I'd like to have it more specific like `errWatch` or maybe `errWatchPrep`.